extensions/loops.py

```
@@ -34,77 +50,77 @@ async def sendSendReadyGiveaways(self) -> None:
     @tasks.loop(seconds=30)
     async def addVoiceUserLoop(self) -> None:
         try:
             await addXpToVoiceUsers(self.bot)  # type: ignore[no-untyped-call]
         except Exception:
-            logging.exception("Error in loop")
+            _log_loop_error("addVoiceUserLoop")
 
     @tasks.loop(seconds=60)
     async def refillAiTokenLoop(self) -> None:
         try:
             await refill_ai_token(self.bot)
```

Flake8 (7.3.0)

[error] undefined name 'refill_ai_token'

(F821)